### PR TITLE
Add tests for some parsers and attendance package

### PR DESCRIPTION
**Pull request by applepiofmyeye**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -530,15 +530,15 @@ Marks the indicated employee as “Absent”, “Late”, or “Present”.
...
+`mark INDEX /atd Attendance_Type` or `mark /n NAME /atd Attendance_Type`
 
 * The attendance type can only be: `ABSENT`, `LATE`, or `PRESENT`.
 * It is not case-insensitive. e.g. you can either enter ABSENT or absent.
 
 Examples:
 
-* `mark n/John /at Absent` marks John as absent.
-* `mark 1 /at Present` marks the first employee in the current list to be present.
+* `mark n/John /atd Absent` marks John as absent.
```
Comment (Kevin-Liusx) [inline docs/UserGuide.md:540]: I think this is change of feature

File: src/main/java/seedu/address/model/person/attendance/AttendanceStorage.java [hunk 2]
```diff
@@ -148,7 +154,11 @@ private int getCount(AttendanceType type, JoinDate joinDate, int numOfLeave) {
      *
      * @return count of number of attendances
      */
-    public int[] getAttendanceReport(JoinDate joinDate, int numOfLeave) {
+    public int[] getAttendanceReport(JoinDate joinDate, int numOfLeave) throws IllegalArgumentException {
+
+        if (numOfLeave < 0) {
+            throw new IllegalArgumentException("Number of leaves cannot be negative!");
+        }
```
Comment (jibtaf) [inline src/main/java/seedu/address/model/person/attendance/AttendanceStorage.java:161]: change of feature

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 3]
```diff
@@ -14,7 +14,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_JOIN_DATE = new Prefix("/jd");
     public static final Prefix PREFIX_SALARY = new Prefix("/s");
     public static final Prefix PREFIX_ANNUAL_LEAVE = new Prefix("/l");
-    public static final Prefix PREFIX_ATTENDANCE_TYPE = new Prefix("/at");
+    public static final Prefix PREFIX_ATTENDANCE_TYPE = new Prefix("/atd");
```
Comment (jibtaf) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:17]: should be /at

**Comment by jibtaf** [comment]
lgtm!

**Pull request merged**